kannada/kannada_kali/kannada_kali.md

```diff
@@ -1 +1,3 @@
-ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
\ No newline at end of file
+ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
+
+ಅದರ ಜೊತೆಗೆ [ಈ]()https://play.google.com/store/apps/details?id=com.sriandroid.justkannada)ಸಾಧನದಿಂದ ಕನ್ನಡದಲ್ಲಿ ಟೈಪ್ ಮಾಡಬಹುದು
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.